docker-compose-generator/docker-fragments/opt-add-bluewallet-lndhub.yml

```diff
@@ -0,0 +1,33 @@
+version: "3.7"
+
+services:
+  bluewallet_lndhub_redis:
+    container_name: bluewallet_lndhub_redis
+    image: "redis:6.2.2-buster@sha256:e10f55f92478715698a2cef97c2bbdc48df2a05081edd884938903aa60df6396"
+    command: "redis-server --requirepass S7bk49NBp29khUci4Zw28vsBviCnYfZ2QLTEsQNizq9ni5ur"
+    restart: "on-failure"
+    stop_grace_period: "1m"
+    init: true
+    volumes:
+      - "bluewallet_lndhub_redisdir:/data"
+
+
+  bluewallet_lndhub_app:
+    container_name: bluewallet_lndhub_app
+    image: "bluewalletorganization/lndhub:v1.3.3@sha256:b99408dfb53f503def405668f5fc4d8185de39a87843d4ea8774544a5df1fe36"
+    user: "0:0"  #need help here as the official docker image from bluewalletorganization uses a user called lndhub, which by default cant read macaroon from BTCPay Server LND.  How to get around this without giving root access?
+    depends_on: [ "bluewallet_lndhub_redis" ]
+    restart: "on-failure"
+    stop_grace_period: "1m"
+    init: true
+    volumes:
+      - "lnd_bitcoin_datadir:/lnd:ro"
+    environment:
+      PORT: "3000"
+      #TOR_URL: "${APP_HIDDEN_SERVICE}"
+      LND_CERT_FILE: "/lnd/tls.cert"
+      LND_ADMIN_MACAROON_FILE: "/lnd/admin.macaroon"
+      CONFIG: '{ "rateLimit": 10000, "postRateLimit": 10000, "redis": { "port": 6379, "host": "bluewallet_lndhub_redis", "family": 4, "password": "S7bk49NBp29khUci4Zw28vsBviCnYfZ2QLTEsQNizq9ni5ur", "db": 0 }, "lnd": { "url": "lnd_bitcoin:10009", "password": ""}}'
```

Comment:
Does the password need to be hardcoded in here?

Reply:
Good question, most Redis installations don't even use a PW.  But I thought it's safer at least to have something, would prevent a random person at least from connecting (if they already have access on your docker network) without authorization.  Happy to know if there is a better way, but most implementations I see do it like this.

Reply:
I think this is fine.

Reply:
Hi @Kukks great that you are also here ;)
I would need some help from someone on including it in services within BTCPay Server interface (if required) and making it work with TOR.  Would you please suggest who could help with that part.  Rest is done and it works well, I am using right now as a custom fragment for a week, without issue.